include/common/mir/input/mir_keyboard_config.h

```diff
@@ -34,14 +34,15 @@ struct MirKeyboardConfig
 {
     MirKeyboardConfig();
     ~MirKeyboardConfig();
-    MirKeyboardConfig(mir::input::Keymap&& keymap);
+    MirKeyboardConfig(std::shared_ptr<mir::input::Keymap> && keymap);
```

Comment:
Do we need an rvalue reference? Surely `MirKeyboardConfig(std::shared_ptr<mir::input::Keymap> keymap)` is at least as efficient.

Reply:
Done